# nanoseconds since epoch
int64 timestamp

# nanoseconds since epoch
int64 vertex_timestamp

# vertex we localize against
uint64 vertex_id

# localization result
LieGroupTransform t_robot_vertex